src/transformers/models/opt/modeling_opt.py

```diff
@@ -802,7 +800,7 @@ def __init__(self, config):
         self.model = OPTModel(config)
 
         # the lm_head weight is automatically tied to the embed tokens weight
-        self.lm_head = nn.Linear(config.word_embed_proj_dim, config.vocab_size, bias=False)
+        self.lm_head = nn.Linear(config.embed_dim, config.vocab_size, bias=False)
```

Comment:
This seems like a fix that isn't linked to the sequence classification head contribution; if that's so, could you split it in a different PR where we can discuss it? Thanks.